src/SamplesApp/UITests.Shared/Windows_UI_Xaml/FocusManager/FocusManager_FocusDirection.xaml

```diff
@@ -15,26 +15,29 @@
 	d:DesignWidth="400">
 
 	<StackPanel>
+		<Button>
+			
+		</Button>
```

Comment:
Why adding this empty button here?

Reply:
_I would assume it is used to steal focus from the textboxes in order to hide the keyboard?_ Since some textboxes may be hidden by the keyboard.
But, yeah... Either use `Content=`  and/or comment to indicate why it is needed, or remove it if it is not needed.

Reply:
Button removed